IviDriverCore/1.0/Spec/IviDriverCore.md

```diff
@@ -0,0 +1,770 @@
+  - [File Versioning](#file-versioning)
+    - [Examples](#examples)
+  - [Driver Conformance](#driver-conformance)
+    - [Driver Verification](#driver-verification)
+      - [Unit Test Procedure](#unit-test-procedure)
+      - [Driver Installation Testing](#driver-installation-testing)
+      - [Driver Buildability Testing](#driver-buildability-testing)
+
+## Overview of the IVI Driver Core Specification
+
+This is the *IVI Driver Core Specification*. It describes requirements common to all IVI Core Drivers, regardless of implementation langage. IVI provides additional specifications that detail the requirements of drivers for specific languages.
+
+Several other documents and specifications are related to this specification. Other documents including language-specific specifications and other extensions are used with this core standard. See the IVI web site at [www.ivifoundation.org](http://www.ivifoundation.org) for a complete list of IVI standards.
+
+These specifications are intended for developers of drivers and tools. Many companies build products and systems around these specifications. This document specifies the requirements for instrument drivers and areas that are open for driver designers to optimize the driver for their particular instrument.
+
+### Specification Conventions
+
+The section describes editorial conventions used in this document and related IVI specifications.
+
+#### Substitutions
+
+This specifications use paired angle brackets to indicate that the text between the brackets is not the actual text to use, but instead indicates the text that is used in place of the bracketed text. Sometimes the meaning is self-evident, and no further explanation is given. The following list includes those that may need additional explanation:
+
+- `<DriverIdentifier>`: This string uniquely identifies the driver. For example, "Agilent34410".
+
+- `<CompanyName>`: The name of the driver vendor (not the instrument manufacturer). For example, "Agilent Technologies, Inc".
+
+- `<RcName>`: The name of a repeated capability. Repeated capabilities may be defined in class specs or by driver developers.
+
+Where it is important to indicate the case of substituted text, casing is indicated by the case of the text between the brackets.
+
+- `<FooBar>` indicates Pascal casing. For example, "IviDmm".
+
+- `<fooBar>` indicates camel casing. For example, "iviDmm".
+
+- `<foobar>` indicates all lower case. For example, "ividmm".
+
+- `<FOOBAR>` indicates all upper case. For example, "IVIDMM".
+
+- `<FOO_BAR>` indicates all upper case with underscores between words. For example, "IVI_DMM".
+
+#### Methods and Properties Nomenclature
+
+This document uses the terms *method* and *property* to refer to elements of the API exported by an IVI driver. Unless specified otherwise, *method* refers generically to C functions and object oriented methods. Similarly, *property* refers to C attributes and object oriented properties.
```

Comment:
```suggestion
This document uses the terms *method* and *property* to refer to elements of the API exported by an IVI driver. Unless specified otherwise, *method* refers generically to C functions and object-oriented methods. Similarly, *property* refers to C attributes and object-oriented properties.
```
Minor nit: I think it should be object-oriented with the hyphen.